api/net/Areas/Editor/Models/Content/ContentModel.cs

```diff
@@ -129,6 +130,12 @@ public class ContentModel : AuditColumnsModel
     /// </summary>
     public DateTime? PublishedOn { get; set; }
 
+    /// <summary>
+    /// get/set - Upload files with content.
+    /// </summary>
+    [DataType(DataType.Upload)]
+    public List<IFormFile>? Files { get; set; }
```

Comment:
At some point we can also upload a file with the original POST and PUT.  Currently we make a separate AJAX request to upload the file.